packages/components/src/internal/components/user/UserDetailsPanel.tsx

```diff
@@ -80,10 +82,17 @@ export class UserDetailsPanel extends React.PureComponent<Props, State> {
                     console.error(error);
                     this.setState(() => ({ userProperties: undefined, loading: false }));
                 });
+
+            try {
+                const groups = await getUserGroups(userId);
+                this.setState(() => ({ groups }));
+            } catch (e) {
+                console.error(e);
```

Comment:
is console.error enough here, or should we render the error somewhere for the user to see?

Reply:
My rationale was that it'd be pretty rare for the getUserGroups() call to fail, and if it does the groups data in the user details panel will just not render—doesn't seem like a substantial enough error to get the red banner treatment, seems like weird UI to render an error output directly in the user details panel, the console.error would be useful for our own debugging, and likely the issue would go away just on refresh.

Reply:
Makes sense.